src/main/scala/net/cakesolutions/CakePlatformPlugin.scala

```diff
@@ -19,30 +25,33 @@ object CakePlatformKeys {
    * We have strict conflict resolution, so it'll catch any mismatches
    * and force you to update your build.sbt
    */
-  object versions {
-    val akka = "2.4.17"
-    val play = PlayVersion.current
+  object Versions {
+    val akka: String = "2.4.17"
+    val play: String = PlayVersion.current
   }
 
-  /** Convenient bundles for depending on platform / core libraries */
-  object deps {
+  /** Convenient bundles for depending on platform/core libraries */
+  object Dependencies {
```

Comment:
Lowercase naming here will cause confusion with package names and break style guides.

I accept that an alternative descriptive name to `Dependencies` might be a better choice, but it really should start with an upper case.